platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -354,33 +321,15 @@ public SwirldsPlatform(@NonNull final PlatformComponentBuilder builder) {
 
         registerAddressBookMetrics(metrics, currentAddressBook, selfId);
 
-        final SyncMetrics syncMetrics = new SyncMetrics(metrics);
         RuntimeMetrics.setup(metrics);
 
-        shadowGraph = new Shadowgraph(platformContext, currentAddressBook, blocks.intakeEventCounter());
-
         final EventConfig eventConfig = platformContext.getConfiguration().getConfigData(EventConfig.class);
 
         keysAndCerts = blocks.keysAndCerts();
 
         EventCounter.registerEventCounterMetrics(metrics);
 
-        final Hash epochHash;
-        if (emergencyRecoveryManager.getEmergencyRecoveryFile() != null) {
-            epochHash = emergencyRecoveryManager.getEmergencyRecoveryFile().hash();
-        } else {
-            epochHash = initialState.getState().getPlatformState().getEpochHash();
-        }
-
         final String swirldName = blocks.swirldName();
-        StartupStateUtils.doRecoveryCleanup(
```

Comment:
`doRecoveryCleanup` is now unused

Reply:
![jail](https://github.com/hashgraph/hedera-services/assets/56973212/aa2cb173-5b3a-4635-bee6-3aa24a3b596d)